homeassistant/components/wled/update.py

```diff
@@ -36,11 +36,33 @@ class WLEDUpdateEntity(WLEDEntity, UpdateEntity):
     )
     _attr_title = "WLED"
 
-    def __init__(self, coordinator: WLEDDataUpdateCoordinator) -> None:
+    def __init__(
+        self,
+        coordinator: WLEDDataUpdateCoordinator,
+        releases_coordinator: WLEDReleasesDataUpdateCoordinator,
+    ) -> None:
         """Initialize the update entity."""
         super().__init__(coordinator=coordinator)
+        self.releases_coordinator = releases_coordinator
         self._attr_unique_id = coordinator.data.info.mac_address
 
+    async def async_added_to_hass(self) -> None:
+        """When entity is added to hass.
+
+        Register extra update listener for the releases coordinator.
+        """
+        await super().async_added_to_hass()
+        self.async_on_remove(
+            self.releases_coordinator.async_add_listener(
+                self._handle_coordinator_update
+            )
+        )
+
+    @property
+    def available(self) -> bool:
+        """Return if entity is available."""
+        return bool(super().available and self.releases_coordinator.last_update_success)
```

Comment:
why the extra `bool`?

Reply:
Welcome to `mypy` 😬 

But in this rare case, it is indeed not needed 😄 Will remove it

Reply:
[d820a51](https://github.com/home-assistant/core/pull/122101/commits/d820a517ee8a3187a7b2fc2ec0337b05bb960ceb)